recipes/r-enmeval/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set version = '0.3.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-enmeval
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ENMeval_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ENMeval/ENMeval_{{ version }}.tar.gz
+  sha256: e7f20c1b4d7ede82a3aca86e3ce3fc4ae46337f812dfc171b499c2558b0183d0
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-dismo
+    - r-doparallel
+    - r-foreach
+    - r-maxnet
+    - r-raster
+  run:
+    - r-base
+    - r-dismo
+    - r-doparallel
+    - r-foreach
+    - r-maxnet
+    - r-raster
+
+test:
+  commands:
+    - $R -e "library('ENMeval')"           # [not win]
+    - "\"%R%\" -e \"library('ENMeval')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=ENMeval
+  license: GPL-2.0-or-later
+  summary: Automatically partitions data into evaluation bins, executes ecological niche models
+    across a range of settings, and calculates a variety of evaluation statistics.  Current
+    version only implements ENMs with Maxent (Phillips et al. 2006) or maxnet (Phillips
+    et al. 2017).
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+    - LICENSE
```

There seems to be no license in the tarball, which is fine.
```suggestion
    - LICENSE
```